src/script/bitcoinconsensus.h

```diff
@@ -54,7 +56,7 @@ enum
 /// txTo correctly spends the scriptPubKey pointed to by scriptPubKey under
 /// the additional constraints specified by flags.
 /// If not NULL, err will contain an error/success code for the operation
-EXPORT_SYMBOL int bitcoinconsensus_verify_script(const unsigned char *scriptPubKey, unsigned int scriptPubKeyLen,
+EXPORT_SYMBOL int bitcoinconsensus_verify_script(const unsigned char *scriptPubKey, unsigned int scriptPubKeyLen, uint64_t amount,
```

Comment:
I thought of my response, but I never sent it. I was going to agree and just clarify (although I think you meant that) that the new version should also work with when the the flags is off so that callers can use a single interface (even if the deprecated one is maintained). But I see that's what has been implemented in https://github.com/sipa/bitcoin/commit/940d74b8731b5fd75cc70357bceb275662915f44 , so great!